### PR TITLE
changed base image for AFNI to fedora:35

### PR DESCRIPTION
**Pull request by stebo85**
https://github.com/ReproNim/neurodocker/issues/519#issuecomment-1465845156

### CODE REVIEW

**Comment by kaczmarj** [comment]
fedora uses the yum/dnf package manager. did this neurodocker command work with `--pkg-manager=apt`?

**Comment by stebo85** [comment]
right, thank you for spotting that. I updated the pkg-manager to yum now

File: docs/user_guide/examples.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ This does not install AFNI's R packages. To install relevant R things, use the f
 
     neurodocker generate docker \
         --pkg-manager apt \
```
Comment (kaczmarj) [inline docs/user_guide/examples.rst:84]:
```suggestion
        --pkg-manager yum \
```

File: docs/user_guide/examples.rst [hunk 2]
```diff
@@ -96,7 +96,7 @@ Beware that this is AFNI's bleeding edge!
...
```
Comment (kaczmarj) [inline docs/user_guide/examples.rst:98]:
```suggestion
        --pkg-manager yum \
```

**Pull request merged**